### ISSUE TITLE
Fix S2094 FP: Allow empty queries

### ISSUE THREAD

**Issue by fraserkyle** [SonarSource/sonar-dotnet#9241]
### Description

It is not an uncommon pattern to have `Query`s with absolutely no members in it when using CQRS design pattern.
In such cases, rule [S2094](https://sonarsource.github.io/rspec/#/rspec/S2094/csharp) raises.

### Repro steps

Consider the following class, which is a totally valid `Query`.

```csharp
public class GetAllUsersQuery {}
```

### Expected behavior

I would expect rule [S2094](https://sonarsource.github.io/rspec/#/rspec/S2094/csharp) not to raise on such classes.

I would suggest ignoring classes that have known `Query` suffix

**Comment by fraserkyle** [comment]
Expands #9063

**Comment by Tim-Pohlmann** [comment]
Hi @fraserkyle, thanks for the report!
I confirm this as an FP.

**Comment by Tim-Pohlmann** [comment]
This will require an RSPEC update.
